joomla/joomla-cms
# [com_redirect] Improve disabled collect url in redirect plugin message

#### Summary of Changes

Following PR https://github.com/joomla/joomla-cms/pull/10688 that improve the link in com_redirect when the redirect plugin is disabled, this PR does similar change to the collect URL disable message.

Some other changes:
- The Collect URL warning is now a notice
- When redirect plugin was enabled AND collect URL option is enabled a notice is displayed. This notice was removed as it's not consistent with the other component list views.
#### Testing Instructions
1. USe latest staging
2. Go to Extensions -> Plugins -> Redirect (System) enable the plugin and disable collect urls
3. Go to Components -> Redirect, you will see the following message
   ![image](https://cloud.githubusercontent.com/assets/9630530/15806666/5690dc98-2b40-11e6-8169-bfc4e627f9d1.png)
4. Apply patch 
5. Go to Components -> Redirect, you will see the following message
   ![image](https://cloud.githubusercontent.com/assets/9630530/15806671/800ff428-2b40-11e6-9435-39d892ae07b4.png)
6. Now enable the collect URL in the redirect plugin, you'll see no message
